Refit.Tests/Refit.Tests.csproj

```diff
@@ -28,51 +28,25 @@
     <Compile Remove="RefitStubs.*.cs" />
   </ItemGroup>
 
-  <Target Name="GenerateRefitStubsFull" Condition="'$(DesignTimeBuild)' != 'true' and '$(MSBuildRuntimeType)' != 'Core' " BeforeTargets="BeforeCompile">
+  <Target Name="GenerateRefitStubs" Condition="'$(DesignTimeBuild)' != 'true' " BeforeTargets="BeforeCompile">
     <PropertyGroup>
-      <OutputStubsFile Condition="'$(TargetFramework)' == 'net461' ">$(MSBuildThisFileDirectory)RefitStubs.Net46.cs</OutputStubsFile>
-      <OutputStubsFile Condition="'$(TargetFramework)' == 'netcoreapp2.1' ">$(MSBuildThisFileDirectory)RefitStubs.NetCore2.cs</OutputStubsFile>
+      <OutputStubsFile>$(MSBuildThisFileDirectory)RefitStubs.$(TargetFramework).cs</OutputStubsFile>
       <RefitParameterFile>$(MSBuildThisFileDirectory)$(IntermediateOutputPath)RefitParams.rsp</RefitParameterFile>
+      <RefitExecutableBase>$(MSBuildThisFileDirectory)../InterfaceStubGenerator.App/bin/$(ConfigurationName)/$(TargetFramework)/InterfaceStubGenerator.App</RefitExecutableBase>
```

Comment:
This may not always match. We don't build the app for every version of the tests we run. For example, we'll want to run .NET Core 3 tests but the app itself will still be for .NET Core 2.1 (as there's no reason to bump it).